app/models/concerns/watchable.rb

```diff
@@ -25,4 +25,22 @@ def notification_title
       "「#{self[:title]}」のイベント"
     end
   end
+
+  def filter_product
+    if watchable_type == 'Product'
+      watchable.body
+    else
+      watchable.description
+    end
+  end
+
+  def time
+    if watchable_type == 'Report'
+      watchable.reported_on
+    elsif watchable.has_attribute?(:published_at) && watchable.published_at
```

Comment:
```suggestion
    elsif watchable.has_attribute?(:published_at) && watchable.published_at?
```

存在確認をするのであればこう書いた方が意図がわかりやすいかも？です〜

Reply:
確かにそうですね:eyes:変更します。